frontend/src/components/drawer/NavigationDrawer.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <v-navigation-drawer
+        :rail="shouldShowRailVersion"
+        :expand-on-hover="shouldShowRailVersion"
+        :model-value="showDrawer"
+        @update:modelValue="this.$emit('hideDrawerEvent')"
+    >
+        <v-list>
+            <v-list-item :prepend-avatar="getUserAvatarImg" title="Użytkownik" subtitle="user@example.com" />
+        </v-list>
+        <v-divider />
+        <div class="buttons-list" v-for="(buttonsKeys, index) in Object.keys(buttons)" :key="buttonsKeys">
+            <v-list density="compact">
+                <v-list-item
+                    v-for="button in buttons[buttonsKeys]"
+                    :key="button.title"
+                    :prepend-icon="button.icon"
+                    :title="button.title"
+                    @click="button.onClick"
+                />
+            </v-list>
+            <v-divider v-if="index < Object.keys(buttons).length - 1" />
+            <!-- If is not a last key -->
+        </div>
+    </v-navigation-drawer>
+</template>
+
+<script>
+import { defineComponent } from "vue";
+import { openModal } from "jenesius-vue-modal";
+import LoadProjectModal from "@/components/modals/menu/LoadProjectModal.vue";
+import SaveProjectModal from "@/components/modals/menu/SaveProjectModal.vue";
+import ShowDebugCodeModal from "@/components/modals/code/ShowDebugCodeModal.vue";
+import { setCurrentThemeInStorage } from "@/javascript/storage/themeStorage";
+import user from "@/img/user.png";
```

Comment:
Zmieniłbym nazwę na userImage, żeby podkreślić, że to jest zdjęcie, a nie obiekt usera